massifs/rootsigner.go

```diff
@@ -85,29 +158,159 @@ func (rs RootSigner) Sign1(coseSigner cose.Signer, keyIdentifier string, publicK
 		return nil, err
 	}
 
-	// We purposefully detach the root so that verifiers are forced to obtain it
+	// We purposefully detach the peaks so that verifiers are forced to obtain it
 	// from the log.
-	state.Root = nil
+	state.LegacySealRoot = nil
+	state.Peaks = nil
+
 	payload, err = rs.cborCodec.MarshalCBOR(state)
 	if err != nil {
 		return nil, err
 	}
+
 	msg.Payload = payload
 
-	return msg.MarshalCBOR()
+	encodable, err := commoncose.NewCoseSign1Message(&msg)
+	if err != nil {
+		return nil, err
+	}
+	return encodable.MarshalCBOR()
+}
+
+// Note: regarding why and how we can pre-sign receipts:
+//
+// A specific advantage of MMR's is that we can pre-sign the protected headers
+// for all receipts we will ever be asked for. The scitt endpoint then only has
+// to copy the pre-signed receipt and *add* the inclusion path it is asked for.
+//
+// Importantly, this allows for self service *privacy preserving*, scitt
+// compatible, receipts based on replicated copies of the log.
+//
+// The most natural place to produce the pre-signed receipts is in the the log
```

Comment:
Is it worth mentioning that the payload of the cose sign 1 message receipt is the accumulator peak for any particular mmr state?

I found that is pretty cool

Reply:
maybe a copy pasta from the comment below?

Reply:
I've updated the comment in signEmptyPeakReceipts to be more explanitory, including highlighting that very important "low update frequency" property for the peaks & proofs